server/sql/subscriptions/getSubscriptionsByUserId.sql

```diff
@@ -0,0 +1,9 @@
+SELECT 
+  name, 
+  nickname, 
+  to_char(due_date, 'YYYY-MM-DD') AS "dueDate", 
+  reminder_days AS "reminderDays", 
+  amount::numeric, 
+  subscription_uuid AS "subscriptionUuid"
```

Comment:
you don't need quotes for all of these aliases:

```suggestion
  to_char(due_date, 'YYYY-MM-DD') AS dueDate,
  reminder_days AS reminderDays,
  amount::numeric,
  subscription_uuid AS subscriptionUuid
```